e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -17,6 +17,13 @@ param(
     [Parameter()]
     [switch] $InstallDependencies,
 
+    # Obsoleted. Recommend docker on WSL2 instead - setup with <repo>/e2e/test/docker/haproxy.sh on WSL.
```

Comment:
You just added it and it is already obsoleted?

Reply:
Adding this to allow existing dev that has docker for Windows to continue to use what they have. Also, we may come back to use this instead of WSL if docker for Windows fix their virtual network issue in the future.